assets/src/js/pro/_acf-blocks.js

```diff
@@ -110,7 +110,7 @@ const md5 = require( 'md5' );
 	 * @return boolean
 	 */
 	function isSiteEditor() {
-		return typeof pagenow === 'string' && pagenow === 'site-editor';
+		return document.querySelectorAll( 'iframe[name="editor-canvas"]' ).length > 0;
```

Comment:
Is this backported 1:1? Is there any other condition that makes this iframe check more accurate than just `wp.data.select('core/edit-site')` to know if we are in the Site Editor?

Reply:
Yep.